src/store/modules/farmer/types.ts

```diff
@@ -11,6 +11,7 @@ export interface Farmer {
   phoneArea: string;
   phoneNumber: string;
   companyName?: string;
+  uuid:string;
```

Comment:
I'm not sure what to do about this. It was removed in your sync branch but we need It. I'm assuming you have it somewhere else I can get it from?